lambda_layers/core_layer/python/core_layer/alembic/versions/35997fffdb9d_add_finish_timestamps_to_closed_reviews.py

```diff
@@ -0,0 +1,26 @@
+"""Add finish timestamps to closed reviews
+
+Revision ID: 35997fffdb9d
+Revises: 7e1207d1a7e1
```

Comment:
Is it a good idea to have two revisions with the same `down_revision` this one and item_types?

Reply:
No https://stackoverflow.com/questions/22342643/alembic-revision-multiple-heads-due-branching-error
Maybe you can move this statement/migration to your branch?